src/ansys/fluent/core/__init__.py

```diff
@@ -155,7 +155,7 @@ def version_info() -> str:
 FLUENT_SHOW_MESH_AFTER_CASE_READ = False
 
 # Whether to write the automatic transcript in Fluent
-FLUENT_AUTOMATIC_TRANSCRIPT = False
+FLUENT_AUTOMATIC_TRANSCRIPT = env_var_to_bool("PYFLUENT_FLUENT_AUTOMATIC_TRANSCRIPT")
```

Comment:
`FLUENT_AUTOMATIC_TRANSCRIPT` is not an environment variable. It's a PyFluent attribute.